FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
these FWCore/* package are included because they are either part of fwlite application set or needed by application set
```
FWCore/Framework included in
  CommonTools/Utils
  DataFormats/CSCRecHit
  DataFormats/FEDRawData
  DataFormats/HcalDigi
  DataFormats/HcalRecHit
  DataFormats/Math
  DataFormats/MuonDetId
  DataFormats/Scalers
  Fireworks/Core
  Fireworks/ParticleFlow

FWCore/ServiceRegistry included in
  CommonTools/Utils
  DataFormats/CSCRecHit

FWCore/PythonUtilities included in
 - part of application set

FWCore/PluginManager included in
  DataFormats/HcalDigi
  DataFormats/HcalRecHit
  DataFormats/Math
  Fireworks/Core

FWCore/PythonParameterSet included in
  FWCore/PythonParameterSet
  PhysicsTools/FWLite
```